### PR TITLE
Set up new Unified Pipeline for Form Recognizer

### PR DESCRIPTION
**Pull request by annelo-msft**
Following [instructions](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/117/Setting-up-a-new-Unified-Pipeline) for setting up a new Unified Pipeline

Adding ci.yml for form recognizer.

### CODE REVIEW

File: sdk/formrecognizer/ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    include:
+    - sdk/formrecognizer/
+
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: template
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Template
```
Comment (pakrym) [inline sdk/formrecognizer/ci.yml]: You also need to change artifacts.

File: sdk/formrecognizer/ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
```
Comment (pakrym) [inline sdk/formrecognizer/ci.yml]: Hmmmmmm...

Comment (annelo-msft) [reply sdk/formrecognizer/ci.yml]: I was quite confused about this.

File: sdk/formrecognizer/ci.yml (hunk 1 above)
Comment (annelo-msft) [reply sdk/formrecognizer/ci.yml]: That got lost in a copy somehow.  Thanks for catching - updated!

**Review by pakrym — APPROVED** [review]

File: sdk/formrecognizer/ci.yml (hunk 2 above)
Comment (weshaggard) [reply sdk/formrecognizer/ci.yml]: LOL... yeah lets call it a deterrent to keep people from playing with these files. We do have a goal to generate these at some point also but just haven't gotten to it yet.

**Pull request merged**